### PR TITLE
Calculating bike-stats

### PR DESCRIPTION
**Pull request by SviatRudik**

### CODE REVIEW

File: src/main/scala/com/scala/aggregator/StatisticalAggregator.scala [hunk 1]
```diff
@@ -17,4 +17,13 @@ class StatisticalAggregator {
     UsageStatistics(result)
   }
 
+  def calculateBikeStats(listOfRides: Seq[Ride]): Seq[((String, Int), Int)] = {
+    val ridesPerBike = listOfRides.groupBy(x => x.bikeId).map {
```
Comment (alexander-belyaev) [inline src/main/scala/com/scala/aggregator/StatisticalAggregator.scala:21]: If you can't find a proper name for a variable, just keep it empty 
_.bikeId

File: src/main/scala/com/scala/aggregator/StatisticalAggregator.scala [hunk 2]
```diff
@@ -17,4 +17,13 @@ class StatisticalAggregator {
...
+      case (bikes, rides) => (bikes, rides)
+    }
+    val durationPerBike = ridesPerBike.map(ride => ride._2.map(duration => duration.duration).sum).toSeq
```
Comment (alexander-belyaev) [inline src/main/scala/com/scala/aggregator/StatisticalAggregator.scala:24]: Use _

File: src/main/scala/com/scala/util/ResultFileCreator.scala [hunk 3]
```diff
@@ -20,4 +21,12 @@ class ResultFileCreator {
     out.close()
   }
 
+  def createBikeStatsFile(listOfBikeStats: Seq[BikeStatistics]) = {
+    val out = new PrintWriter(NameOfBikeStatsFile, "UTF-8")
+    out.println("Bike id/amount of rides/duration")
```
Comment (alexander-belyaev) [inline src/main/scala/com/scala/util/ResultFileCreator.scala:26]: Hm, we were talking about CVS file, why you're using / as a separator?

**Review by alexander-belyaev — CHANGES_REQUESTED** [review]

**Comment by alexander-belyaev** [comment]
a) add some content inside Readme.md
b) stop committing target file to github

File: src/main/scala/com/scala/util/ConverterUtil.scala [hunk 4]
```diff
@@ -29,4 +29,11 @@ object ConverterUtil {
     MemberType.withName(stringMemberType)
 
   }
+
+  def convertToBikeStatistics(ridePerBikeZipDuration: Seq[((String, Int), Int)]): Seq[BikeStatistics] = {
+    val bikeStats = ridePerBikeZipDuration.map { el =>
+      BikeStatistics(el._1._1, el._1._2, el._2)
+    }
+    bikeStats
```
Comment (VladimirBaklan) [inline src/main/scala/com/scala/util/ConverterUtil.scala:37]: You can make it more readable by using pattern matching:
```
(replace nameOfTheAttributeN by the name of the variable)
ridePerBideZipDuration.map { case ((nameOfTheAttribute, nameOftheAttribute2), nameOfTheAttribute3) =>
BikeStatistics(nameOfTheAttribute, nameOftheAttribute2, nameOfTheAttribute3)
}
```

**Review by alexander-belyaev — APPROVED** [review]

**Review by VladimirBaklan — APPROVED** [review]

**Pull request merged**